soundio/soundio.h

```diff
@@ -18,14 +18,18 @@
 #define SOUNDIO_EXTERN_C
 #endif
 
-#if defined(_WIN32)
-#if defined(SOUNDIO_BUILDING_LIBRARY)
-#define SOUNDIO_EXPORT SOUNDIO_EXTERN_C __declspec(dllexport)
+#if defined(SOUNDIO_STATIC_LIBRARY)
```

Comment:
Where is SOUNDIO_STATIC_LIBRARY defined? Would the user define that before including soundio.h?